bindings/src/query_resp.rs

```diff
@@ -694,6 +694,67 @@ pub struct LeveragelpPoolsResponse {
     pub pool: Vec<LeveragelpPool>,
     pub pagination: Option<PageResponse>,
 }
+
+#[cw_serde]
+pub struct LeveragelpOpenEstResponse {
+    position_size: Int128,
+    weight_balance_ratio: Decimal,
+    borrow_fee: Decimal,
+}
+
+#[cw_serde]
+pub struct LeveragelpCloseEstResponse {
```

Comment:
This is wrong. See https://github.com/elys-network/elys/pull/544/files#diff-608a4d4096e8cad7b23065a30151bc5cb22c87e505600789c15cd71c3a0c8d11R229